### ISSUE TITLE
Poor accuracy when trying to run inference on single and short expressions

### ISSUE THREAD

**Issue by BrianPark314** [breezedeus/Pix2Text#95]
Hello! I want say that I appreciate your work very much. I am building an app that can recognise highschool level math expressions, and I ran into some issues.

1. As pointed out in the title, the model is poor at recognising one or two length expressions (simple expressions) such as 1, 0, a, b, 10, 11, etc. via the recognise_formula method. It may seem obvious but it struggles a lot with ones and zeros. Furthermore the output is sometimes very weird when I try to inference 1, like \overbrace{1}. 
2. I tried to resolve this issue by manipulating allowed tokens to be generated by passing generationConfig parameter on generation (for which I plan on writing a PR), which solves the issue of unused vocab but still doesn't help improving the accuracy of shor expression.
3. As a matter of fact, I had to look into the vocab settings of the transformer and I found out that the encoder vocab has many strange and duplicated characters, and disallowing them affects the model's capability to generate math expressions correctly. 

I hope you can bring insight on how to work around this issue. It would be even greater help if I can figure out how to get low-level access to the model so that I can retrain it or fine-tune it for my purpose.

**Comment by breezedeus** [comment]
Can you give a couple of sample images and your code calling it?

**Comment by BrianPark314** [comment]
``
pix2Text_model.recognize_formula(
            imgs=img,
            resized_shape=768,
            return_text=False,
        )
``
This is my code and here are some example images:
![image](https://github.com/breezedeus/Pix2Text/assets/125226023/9c046ec1-9ff9-4742-9d81-4e8afe2ba6f8)
result: \\mathbf{\\delta}
![image](https://github.com/breezedeus/Pix2Text/assets/125226023/8c8948d8-e310-49da-a362-f5d05e89529c)
result: \\lfloor D
![image](https://github.com/breezedeus/Pix2Text/assets/125226023/035eaf0e-d667-4a05-9ebb-f2f8563d2cc5)
result: \\{~ | ~ |
![image](https://github.com/breezedeus/Pix2Text/assets/125226023/131473e9-0fd6-46fa-8f9b-1410c0866540)
result: ( a x+b )

**Comment by breezedeus** [comment]
Hi, it's true that Pix2Text's MFR model is not specifically optimized for handwritten text images. If you want to continue training with your own data, you'll have to use the PyTorch version of the MFR model. The current paid product that includes the PyTorch version is: [[Enterprise Pro Subscription] Pix2Text (P2T) V1.0 Models: MFR + MFD + CnOCR](https://ocr.lemonsqueezy.com/buy/54ecbde4-40e5-47cd-919d-c6ac6201523a). It should be suitable if you are using it for business use, but it may be too expensive for personal use.
